packages/package-values.yaml

```diff
@@ -23,7 +23,7 @@ managementPackageRepository:
     - name: featuregates
```

Comment:
nit: This file has `.yaml` extension, while kbld config file has `.yml` extension. Can we make it consistent and have it `.yaml`?